src/arrayMethodJoin.js

```diff
@@ -4,8 +4,29 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let str = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const item = this[i];
+
+      if (typeof item === 'string'
+        || typeof item === 'number'
+        || typeof item === 'boolean') {
```

Comment:
you can check your item only for null or undefined, because we can pass even function here.
<img width="414" alt="Screenshot 2023-03-23 at 9 35 19 AM" src="https://user-images.githubusercontent.com/60295799/227134591-7949813b-c404-4328-82a9-9f9a730b26bf.png">

```suggestion
      if (item !== null && item !== undefined) {
        str += item
      }
```